### PR TITLE
Add divide function

### PR DESCRIPTION
**Pull request by derrwei**
add divide

### CODE REVIEW

File: divide.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```
Comment (smbrealy) [inline divide.py]: z should be y

**Review by smbrealy — COMMENTED** [review]
see comment on divide.py file

File: divide.py (hunk 1 above)
Comment (OliverPreston) [inline divide.py]: z should be y

**Review by smbrealy — CHANGES_REQUESTED** [review]
please update code

**Review by OliverPreston — CHANGES_REQUESTED** [review]

**Comment by smbrealy** [comment]
looks good

**Review by smbrealy — APPROVED** [review]

**Review by OliverPreston — APPROVED** [review]

**Pull request merged**